style-guide.md

```diff
@@ -50,116 +50,133 @@ This section of the document provides guidelines on best practices, language usa
+- Use Chicago title-style capitalization for titles and headings, which capitalizes the first word plus all other significant words.
+- Capitalize product names.
+- Avoid unnecessary capitalization; before you capitalize a word, think about why and if it should be capitalized.
 
 ### Table Formatting
 
-- Use tables to represent sets of related pieces of data in a structured way
-- Table headers and values should be centered
-- Tables should be formatted. You can use a tool to format the tables, like the [Markdown Table Formatter VSCode extension](https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter)
+- Use tables to represent sets of related pieces of data in a structured way.
+- Table headers and values should be centered.
+- Tables should be formatted. You can use a tool to format the tables, like the [Markdown Table Formatter VSCode extension](https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter).
 
 ### List Formatting
 
-- Use ordered (numbered) lists for a sequence of steps
-- Use unordered (bulleted) lists for items that are non-sequential and can be read or completed in any order
-- For description lists, use the following formatting: `**term** - description`
-  - Put the term in bold
-  - Use a dash (`-`) between the term and the description
-  - Do not capitalize the first word in the description, unless it's an acronym or product name that should be capitalized
-- Do not add punctuation at the end of each list item
+- Use ordered (numbered) lists for a sequence of steps.
+- Use unordered (bulleted) lists for items that are non-sequential and can be read or completed in any order.
+- For description lists, use the following formatting: `**Term**: Description.`
+  - Put the term in bold.
+  - Capitalize the term.
+  - Use a colon (`:`) between the term and the description.
+  - Capitalize the first word of the description, unless the first word is a code element (e.g., a function name, file name, or variable).
+  - If the description introduces a nested list, restructure the sentence so that only one colon is present—at the end.
+- Add punctuation at the end of each list item if the item is a full sentence or a complete idea.  
+  Example:  
+  _You can do any of the following by using the API:_
+  - Create an item.
+  - Replace one item with another.
+  - Update an item.
+  - Delete an item.
+- Do not add punctuation at the end of list items if the item consists of a single word or a short piece of code.  
+  Example:  
+  _The API supports the following actions:_
+  - Create  
+  - Replace  
+  - Update  
+  - Delete
+- Maintain consistent grammar structure across all items in a list (e.g., all items should start with a verb, or all should be noun phrases).
+- For more information about list formatting, refer to the [Google Developer Style Guide on Lists](https://developers.google.com/style/lists#types-of-lists).
```

Comment:
I would emphasize this and state that this list here is not a comprehensive guideline for formatting lists. Like half of it is here, there's more on the Google developer style guide. We are no longer straying from the Google style guide, so really like if they want to find everything in one place, just go there